sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -8,437 +8,215 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
 
```

Comment:
Just to confirm, Changes in this file is same as what was in 1946?

Reply:
Correct, I did not change that file